perf/perf_sdt_probe.py

```diff
@@ -54,10 +54,10 @@ def add_library(self):
         for line in str(self.libpthread).splitlines():
             if re.search('libpthread.so', line, re.IGNORECASE):
                 if 'lib64' in line:
-                    self.libpthread = line.split(" ")[7]
+                    self.libpthread = line.split("=>")[1]
```

Comment:
What about on old distro

Reply:
> What about on old distro

I have tested with the change, works fine on older 8.x 9.x and SLES15 SPx distros.